src/components/Category.jsx

```diff
@@ -0,0 +1,95 @@
+import adidasShoes from "../assets/adidas-shoes.jpg";
```

Comment:
You can keep these urls in the backend in the model

Reply:
These are suggestions. You can proceed with merge for now.

Reply:
Will do the cloudinary part in another branch separately as i progress further with this project....... thank you for your feedback. 🌻

Reply:
please approve the changes

Reply:
Already approved.